### PR TITLE
Support order details universal links

### PR DESCRIPTION
**Pull request by nikola-mladenovic**
Should be merged after PR #144

### CODE REVIEW

**Comment by nikola-mladenovic** [comment]
This implementation supports both orders and reservations; meaning if the link contains an order ID from a reservation, the reservation confirmation screen will be shown, or the classic order details screen otherwise.

**Comment by nikola-mladenovic** [comment]
TestFlight build # 79

File: Sunrise/AppDelegate.swift [hunk 1]
```diff
@@ -63,6 +63,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
                 AppRouting.switchToProductDetails(for: sku)
                 return true
 
+            // Orders (e.g https://demo.commercetools.com/en/user/orders/87896195?)
```
Comment (cneijenhuis) [inline Sunrise/AppDelegate.swift]: Does the URL really end with a `?`?

**Review by cneijenhuis — COMMENTED** [review]
Should we give some sort of error message if the order number does not exist/does not belong to this customer?

File: Sunrise/AppDelegate.swift (hunk 1 above)
Comment (nikola-mladenovic) [reply Sunrise/AppDelegate.swift]: It actually does - that is what I get on demo.commercetools.com when viewing an order.

Comment (cneijenhuis) [reply Sunrise/AppDelegate.swift]: I got a URL without it, but let's keep it as it works both ways 😉

**Comment by nikola-mladenovic** [comment]
Added error message if the SKU in the PDP link is not found (related to #144 )

**Comment by nikola-mladenovic** [comment]
Added error message if the order number in the order link is not found

**Review by cneijenhuis — APPROVED** [review]

**Pull request merged**